### PR TITLE
Created README and CONTRIBUTING files in Japanese

### PR DESCRIPTION
**Pull request by lg970325**
I translated README.md and CONTRIBUTING.md into Japanese. Saved as README_JP.md and CONTRIBUTING_JP.md respectively. 
I hope Japanese users will be able to use this project more conveniently. 
Thank you.

### CODE REVIEW

File: README_JP.md [hunk 1]
```diff
@@ -0,0 +1,317 @@
+English(README.md) | [简体中文](README_CN.md) | [Deutsch](README_GE.md) | 日本語
```
Comment (goswami-rahul) [inline README_JP.md]: Please also make English README.md as a link in this line by -- 
```
[English](README.md) | [简体中文](README_CN.md) | [Deutsch](README_GE.md) | 日本語
```

Comment (lg970325) [reply README_JP.md]: Ok. I will add your requirements and pull request again.

Comment (lg970325) [reply README_JP.md]: I added the link to Japanese README file in other README_XX.md files.

Comment (goswami-rahul) [reply README_JP.md]: Please also add the link to English README file in your `README_JP.md` file.

Comment (lg970325) [reply README_JP.md]: Added :)

**Pull request merged**